Packages/Status/Sources/Status/Row/StatusRowView.swift

```diff
@@ -455,23 +455,35 @@ public struct StatusRowView: View {
   private var trailingSwipeActions: some View {
     if preferences.swipeActionsStatusTrailingRight != StatusAction.none {
       makeSwipeButton(action: preferences.swipeActionsStatusTrailingRight)
-        .tint(theme.tintColor)
+        .tint(
+          preferences.swipeActionsStatusTrailingRight == .boost ? theme.tintColor :
+            preferences.swipeActionsStatusTrailingRight == .favorite ? .yellow : preferences.swipeActionsStatusTrailingRight == .bookmark ? .pink : .gray
+        )
     }
     if preferences.swipeActionsStatusTrailingLeft != StatusAction.none {
       makeSwipeButton(action: preferences.swipeActionsStatusTrailingLeft)
-        .tint(.gray)
+        .tint(
```

Comment:
This should be refactored in some function to make it shorter and easier to read.

Reply:
@Dimillian Please take a look :)